packages/router/__tests__/matcher/pathParser.spec.ts

```diff
@@ -619,8 +619,59 @@ describe('Path parser', () => {
     })
 
     it('can not match the end', () => {
-      matchParams('/home', '/home/other', null, { end: true })
```

Comment:
Was this test removed by accident? It would be great to have it

Reply:
It was intentional, but I've just pushed another commit to bring it back. This also adds extra tests to check that `end: true` works as expected in various simple cases, either when passed explicitly or when left as the default.